internal/validator/default/helpers.go

```diff
@@ -0,0 +1,78 @@
+package defaultvalidator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/bitcoin-sv/arc/internal/validator"
+	"github.com/libsv/go-bt/v2"
+)
+
+var errParentNotFound = errors.New("parent transaction not found")
+
+func extendTx(ctx context.Context, w validator.TxFinderI, rawTx *bt.Tx) error {
```

Comment:
The same as for `BeefValidator`, why not make it the method of `DefaultValidator` and access its `TxFinder`?